packages/next/src/bin/next.ts

```diff
@@ -56,6 +55,20 @@ if (!foundCommand && args['--help']) {
   process.exit(0)
 }
 
+// Check supported Node.js version first
+if (
+  semver.lt(process.versions.node, process.env.__NEXT_REQUIRED_NODE_VERSION!)
```

Comment:
It'd be best to move this at the top of the file, the start time needs to include the time it takes to measure the import time